### PR TITLE
DCOS-8058: Add client side network validation

### PR DESCRIPTION
**Pull request by orlandohohmeier**
---
ℹ️  You will see the validation errors only on the second click or after switching the tabs. This is know issue which will hopefully be fixed with https://github.com/mesosphere/reactjs-components/pull/303.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

**Comment by jfurrow** [comment]
@orlandohohmeier I pushed some CSS which fixes the alignment issue:

File: src/js/schemas/service-schema/Networking.js [hunk 1]
```diff
@@ -133,7 +135,20 @@ const Networking = {
         properties: {
           lbPort: {
             title: 'LB Port',
-            type: 'number'
+            type: 'number',
+            externalValidator: function ({networking}, definition) {
+              const {[definition.name]:port}= networking;
```
Comment (philipnrmn) [inline src/js/schemas/service-schema/Networking.js]: nit: missing space before `=`

**Comment by philipnrmn** [comment]
Code generally looks good, and I was also able to force the errors to appear.

File: src/js/schemas/service-schema/Networking.js [hunk 2]
```diff
@@ -133,7 +135,20 @@ const Networking = {
         properties: {
           lbPort: {
             title: 'LB Port',
-            type: 'number'
+            type: 'number',
+            externalValidator: function ({networking}, definition) {
+              const {[definition.name]:port} = networking;
```
Comment (kennyt) [inline src/js/schemas/service-schema/Networking.js]: is it possible to have a space after `:`?

**Comment by kennyt** [comment]
the code looks good to me 👍

**Comment by Poltergeist** [comment]
LGTM :shipit:

**Pull request merged**